csharp/Yoga/Facebook.Yoga.Native.msbuildproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.Build.NoTargets/3.7.0">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <PackageId>Facebook.Yoga.Native.$(RuntimeIdentifier)</PackageId>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="NuGetizer" Version="0.9.1" />
+  </ItemGroup>
+
+  <Target Name="BuildNative" BeforeTargets="CoreBuild" Returns="@(NativeFileFullPath)">
+    <Exec Command="cmake -S . -B build -DCMAKE_INSTALL_PREFIX=binnative &amp;&amp; cmake --build build --target install --config $(Configuration)" />
```

Comment:
```suggestion
    <Exec Command="cmake -S . -B build -DCMAKE_INSTALL_PREFIX=binnative -DCMAKE_BUILD_TYPE='$(Configuration)' &amp;&amp; cmake --build build --target install --config $(Configuration)" />
```

Reply:
I remember writing this somewhere earlier but might have closed the tab. Apologies if it is a duplicate.